### PR TITLE
Complete secrets mounting for auto dependency update

### PR DESCRIPTION
**Pull request by chxchx**
**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by chxchx** [comment]
/assign @sebastienvas
PTAL

File: prow/config.yaml [hunk 1]
```diff
@@ -701,10 +701,18 @@ periodics:
...
+        name: e2e-testing-kubeconfig
     volumes:
+    - name: oauth
+      secret:
+        secretName: oauth-token
     - name: service
       secret:
         secretName: service-account
-    - name: oauth
+    - name: e2e-testing-kubeconfig
```
Comment (sebastienvas) [inline prow/config.yaml]: You don't need e2e-testing-kubeconfig

**Review by sebastienvas — CHANGES_REQUESTED** [review]

File: prow/config.yaml (hunk 1 above)
Comment (chxchx) [reply prow/config.yaml]: Done

**Review by sebastienvas — APPROVED** [review]

**Pull request merged**